packages/protocol/test/governance/network/governance.ts

```diff
@@ -1500,6 +1500,113 @@ contract('Governance', (accounts: string[]) => {
     })
   })
 
+  describe('#revokeVotes()', () => {
+    const proposalId = 1
+    const index = 0
+    const value = VoteValue.Yes
+    beforeEach(async () => {
+      await governance.propose(
+        [transactionSuccess1.value],
+        [transactionSuccess1.destination],
+        // @ts-ignore bytes type
+        transactionSuccess1.data,
+        [transactionSuccess1.data.length],
+        descriptionUrl,
+        // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
+        { value: minDeposit }
+      )
+      await timeTravel(dequeueFrequency, web3)
+      await governance.approve(proposalId, index)
+      await timeTravel(approvalStageDuration, web3)
+      await mockLockedGold.setAccountTotalLockedGold(account, weight)
+      await governance.vote(proposalId, index, value)
+    })
+
+    it('should return true', async () => {
+      const success = await governance.revokeVotes.call()
+      assert.isTrue(success)
+    })
```

Comment:
I'm not sure how useful this test is

Reply:
This is to verify that user knows that all votes are revoked by function returning true.

Reply:
`call` doesn't alter state but I see your point
not sure how familiar you are with the web3 landscape but this might be a good resource https://bitsofco.de/calling-smart-contract-functions-using-web3-js-call-vs-send/

Reply:
I have removed this test case as it will not modify the state. I wasn't aware of it and I copied it from vote tests.